packages/core/src/components/essentials/modal.ts

```diff
@@ -23,13 +23,37 @@ export class Modal extends Component {
 	}
 
 	handleShowModal = () => {
+		this.modal
+			.attr('data-modal', true)
+			.attr('class', 'cds--modal cds--modal')
```

Comment:
why `cds--modal` twice?

Reply:
<img width="1055" alt="image" src="https://github.com/carbon-design-system/carbon-charts/assets/76566868/1f03560e-40ae-4f8d-9e1e-8e1046b535bf">

code already has been that way before the change was made, I'll remove the extra `cds--modal` and make a PR again